components/admin_console/system_users/list/system_users_list.jsx

```diff
@@ -21,6 +21,7 @@ import SystemUsersDropdown from '../system_users_dropdown';
 export default class SystemUsersList extends React.PureComponent {
     static propTypes = {
         users: PropTypes.arrayOf(PropTypes.object),
+        teams: PropTypes.arrayOf(PropTypes.object),
```

Comment:
Nice catches here with the missing fields :)

Reply:
Thanks @jespino - have fixed this in the latest commit.